Automattic/wp-calypso
# Agencies: Enable SMS and multiple email notifications in Horizon

This PR enables the feature flags for SMS input and multiple email input for downtime notifications in Jetpack Cloud, but _only_ for the Jetpack Cloud Horizon environment (i.e., Calypso Live).

## Proposed Changes

* In `jetpack-cloud-horizon.json`, change the following flags from false to true:
  * `jetpack/pro-dashboard-monitor-sms-notification`
  * `jetpack/pro-dashboard-monitor-multiple-email-recipients`

## Testing Instructions

1. Visit the Jetpack Cloud Calypso Live branch associated with this PR.
2. In the Dashboard, verify that SMS notification input and multiple email input are enabled by default.
3. Verify that in other environments, existing behavior is preserved (i.e., disabled in every environment except for local Jetpack Cloud development).
